content/docs/querying/api.md

```diff
@@ -71,6 +71,7 @@ URL query parameters:
 
 - `query=<string>`: Prometheus expression query string.
 - `time=<rfc3339 | unix_timestamp>`: Evaluation timestamp. Optional.
+- `timeout=<duration>`: Evaluation timeout. Optional. Defaults to value of `-query.timeout` flag.
```

Comment:
As an admin of a Prometheus server, I'd want this as a hard limit to avoid queries of sloth.